apps/frontend/src/network/users/Outputs.tsx

```diff
@@ -1,5 +1,119 @@
-import { UserProfileOutputs } from '@asap-hub/react-components';
+import { isEnabled } from '@asap-hub/flags';
+import {
+  ResearchOutputsSearch,
+  ProfileCardList,
+  ComingSoon,
+  SharedResearchList,
+} from '@asap-hub/react-components';
+import { network } from '@asap-hub/routing';
+import React from 'react';
+import { usePagination, usePaginationParams, useSearch } from '../../hooks';
+import { useResearchOutputs } from '../../shared-research/state';
+import { SearchFrame } from '../../structure/Frame';
 
-const Outputs = UserProfileOutputs;
+type OutputsListProps = {
+  searchQuery: string;
+  filters: Set<string>;
+  userId: string;
+};
+
+type OutputsProps = {
+  userId: string;
+};
+
+const OutputsList: React.FC<OutputsListProps> = ({
+  searchQuery,
+  filters,
+  userId,
+}) => {
+  const { currentPage, pageSize, isListView, cardViewParams, listViewParams } =
+    usePaginationParams();
+
+  const result = useResearchOutputs({
+    searchQuery,
+    filters,
+    currentPage,
+    pageSize,
+    userId,
+  });
+  const { numberOfPages, renderPageHref } = usePagination(
+    result.total,
+    pageSize,
+  );
+  return (
+    <SharedResearchList
+      researchOutputs={result.items}
+      numberOfItems={result.total}
+      numberOfPages={numberOfPages}
+      currentPageIndex={currentPage}
+      renderPageHref={renderPageHref}
+      isListView={isListView}
+      cardViewHref={
+        network({}).users({}).user({ userId }).outputs({}).$ + cardViewParams
+      }
+      listViewHref={
+        network({}).users({}).user({ userId }).outputs({}).$ + listViewParams
+      }
+    />
+  );
+};
+
+const Outputs: React.FC<OutputsProps> = ({ userId }) => {
+  const {
+    filters,
+    searchQuery,
+    toggleFilter,
+    setSearchQuery,
+    debouncedSearchQuery,
+  } = useSearch();
+
+  return (
+    <article>
+      <ProfileCardList>
```

Comment:
Do we need to use this? Profile card list is mainly about offering the floating pencils

Reply:
it was to maintain the same spacing as the rest of the page